rllab/envs/gym_util/env_util.py

```diff
@@ -0,0 +1,37 @@
+import gym
```

Comment:
this can be located in rllab.envs.util / rllab/envs/util.py

No need for the long import path.